control/strategy/pool.go

```diff
@@ -297,6 +298,20 @@ func (p *pool) Kill(id uint32, reason string) {
 	}
 }
 
+// Kill all instances of a plugin
+func (p *pool) KillAll(reason string) {
+	for _, rp := range p.plugins {
+		rp.Stop(reason)
```

Comment:
I don't see how that would work, @IRCody; the arguments to the `SelectAndKill(...)` method are a string providing the task ID we're removing instances for and a string providing the reason; the `rp` value you're showing in your snippet, above (even if I did change the for back to `for _, rp := range p.plugins { ... }` is an AvailablePlugins interface, not a string.  Thoughts on how you'd correct that type mismatch?

(Note; the `rp` value in the `SelectAndKill(...)` method comes from the `Remove(...)` call, which depends on having a task ID to use to select which plugin instances we're removing, and we don't have one of those...)